resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/AbstractReactiveRepositoryRedisAdapter.java

```diff
@@ -36,9 +38,11 @@
 import uk.gov.gchq.palisade.service.resource.exception.ReactiveRepositoryReflectionException;
 
 import java.lang.reflect.Field;
+import java.time.Duration;
 import java.util.Arrays;
 import java.util.LinkedList;
 import java.util.function.Function;
+import java.util.stream.Collectors;
 
 /**
  * A class that implements a {@link ReactiveCrudRepository} for Redis backing store
```

Comment:
Some more documentation may be needed here, just describing what the keys <-> values look like in redis and what (redis) types those values are

Reply:
JavaDoc updated